jobserver/models/outputs.py

```diff
@@ -346,6 +346,18 @@ def approve(self, *, user, now=None):
         self.decision = self.Decisions.APPROVED
         self.save(update_fields=["decision_at", "decision_by", "decision"])
 
+    @classmethod
+    @transaction.atomic()
+    def create_from_files(self, *, files, workspace, user):
```

Comment:
I think that convention for a classmethod constructor would be to use `cls` rather than `self` as first parameter, then use `cls.objects.create(...)`, to make it work with inheritance.

Now, if we're inheriting from models, we've probably done something wrong, tbh.  But a `@classmethod` with `self` just looks a bit odd to me?